modules/storages/spec/features/admin_storages_spec.rb

```diff
@@ -107,7 +107,7 @@
         expect(page).to be_axe_clean.within '#content'
 
         # Show Add storage button
-        expect(page).to have_selector :button, 'Storage', aria: { label: 'Add new storage' }
+        expect(page).to have_button 'Storage', aria: { label: 'Add new storage' }
```

Comment:
Didn't know matchers for node attributes could be specified this way just as in views, neat! 🤌